.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,24 @@
+name: AutoMerge 💥
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  hello_world_job:
+    runs-on: ubuntu-latest
+
+    name: Approve and ask for merge
+
+    if: ${{ github.actor == 'dependabot[bot]' }}
```

Comment:
This check is also done inside the action's code, but doing it here allows to totally skip the job.